src/commands/ReminderCommand.ts

```diff
@@ -188,33 +246,50 @@ class ReminderCommand extends Command {
 
 const setMinutes = new SlashCommandIntegerOption()
 	.setName("minutes")
-	.setDescription("Set the minutes.")
+	.setDescription("Set the minutes. (60 seconds)")
 	.setMinValue(0)
-	.setMaxValue(60)
+	.setMaxValue(1440)
```

Comment:
why have arbitary max values at all

Reply:
The max values are always the smallest unit of the unit after the next 
A day has 1440 minutes 
A month has 720 hours 
I think these are nice boundaries 🤷‍♀️